loops.js

```diff
@@ -51,3 +71,14 @@
 // >> 9
 // ==========================================
 
+for (let i = 0 ; i < 10 ; i++){
+    if (i <= 2){
+        console.log(i);
```

Comment:
Probeer consistent te zijn met gebruik van ";". Gebruik ze overal of gebruik ze niet.